redbeams/src/main/java/com/sequenceiq/redbeams/flow/redbeams/start/RedbeamsStartFlowConfing.java

```diff
@@ -27,9 +29,9 @@ public class RedbeamsStartFlowConfing extends AbstractFlowConfiguration<Redbeams
 
     private static final List<Transition<RedbeamsStartState, RedbeamsStartEvent>> TRANSITIONS =
             new Transition.Builder<RedbeamsStartState, RedbeamsStartEvent>().defaultFailureEvent(REDBEAMS_START_FAILED_EVENT)
-                    .from(INIT_STATE).to(START_DATABASE_SERVER_STATE).event(REDBEAMS_START_EVENT)
-                    .defaultFailureEvent()
-                    .from(START_DATABASE_SERVER_STATE).to(REDBEAMS_START_FINISHED_STATE).event(START_DATABASE_SERVER_FINISHED_EVENT)
+                    .from(INIT_STATE).to(START_DATABASE_SERVER_STATE).event(REDBEAMS_START_EVENT).defaultFailureEvent()
+                    .from(START_DATABASE_SERVER_STATE).to(CERT_ROTATE_STATE).event(REDBEAMS_CERT_ROTATE_EVENT).defaultFailureEvent()
```

Comment:
`START_DATABASE_SERVER_STATE` used to have `.failureEvent(START_DATABASE_SERVER_FAILED_EVENT)`, so `defaultFailureEvent()` will not be appropriate here.